src/components/Shared/OTPPhoneVerification.js

```diff
@@ -78,20 +69,16 @@ const OTPPhoneVerification = ({
         record: JSON.stringify(data)
       })
         .then(res => {
-          // console.log(res)
-          // if (res.status === 303) {
-          //   // Follow the redirection with a new GET request
-          //  alert(res.error)
-          // } else {
-          //   formValidation.setFieldValue('OTPVerified', true )
-          // }
-          // formValidation.setFieldValue('OTPVerified', true )
-          // setError(res.error)
+          formValidation.setFieldValue('OTPVerified', true)
```

Comment:
you set them in the form but are they set from the backend ? 
I prefer if you request get2KYC in case of other fields added in the future so we won't have to worry about assigning them here also instead of hardcoding the values like that.

Reply:
it can solve it after solve otpVerfication to get true result